site/e2e/helpers.ts

```diff
@@ -49,6 +50,7 @@ export const createWorkspace = async (
   templateName: string,
   richParameters: RichParameter[] = [],
   buildParameters: WorkspaceBuildParameter[] = [],
+  useExternalAuthProvider: string | undefined = undefined,
```

Comment:
I think that with the new parameter, the function is getting a little unwieldy, and if you're looking at things from the consumer side, it's hard to tell which of the five arguments correspond to what

Maybe this could be refactored to an object, so that we have sort of have named arguments?
```ts
type CreateWorkspaceInputs = Readonly<{
  // Required properties
  page: Page;
  templateName: string;
  
  // Optional properties
  richParameters?: readonly RichParameter[];
  buildParameters?: readonly WorkspaceBuildParameter[];
  externalAuthProvider?: string;
}>
```

Reply:
I agree in principle, but I think 5 params is still manageable and not worth refactoring the 26 usages of this function IMHO. Happy to discuss.